localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ necromancy_group:0 "Necroism"
+ necromancy_group_adj:0 "Necrotic"
+ necromancy_group_adherent:0 "Necroist"
+ necromancy_group_adherent_plural:0 "Necroists"
+ necromancy_group_desc:0 "A group of numerous cults praising Death. $necromancy_group_adherent_plural$ master necromantic magic allowing them to contact with the Other Side."
+ 
+ death_god:0 "Mord'truar"
+ death_god_adj:0 "Mord'truari"
+ death_god_adherent:0 "Mord'truari"
+ death_god_adherent_plural:0 "Mord'truari"
+ death_god_desc:0 "A death cult serving the most powerful necromancer known as the Lich King. The Lich King and $death_god_adherent_plural$ have a strong connection with the Shadowlands. They carve mystical runes on armor and weapons that enhance their abilities."
+ 
+ maldraxxi:0 "Maldraxxi"
+ maldraxxi_adj:0 "Maldraxxi"
+ maldraxxi_adherent:0 "Maldraxxi"
+ maldraxxi_adherent_plural:0 "Maldraxxi"
+ maldraxxi_desc:0 "A death cult drawing powers from the dimension they call Maldraxxus. Their necromancers are known for creating abominations, animated creatures comprised of multiple dead limbs and body parts."
+ 
+ throsic:0 "Throsi"
+ throsic_adj:0 "Throsic"
+ throsic_adherent:0 "Throsic"
+ throsic_adherent_plural:0 "Throsi"
+ throsic_desc:0 "Many spirits of the fallen Drust have been banished into an offshoot plane within the Emerald Nightmare that they later named Thros, where they cultivated a potent form of death magic with bastardized druidic elements. This form of magic resulted from numerous attempts to bewitch the souls of fallen Drust into constructs to use against human settlers. Following this, the Throsic form of necromancy prefers binding the souls of the dead into durable constructs, while the traditional form of corpse reanimation is rarer. Its influence towards the wildlife and the living is usually perpetuated by the local witch covens. The most loyal of Throsian witches can also ascend into Matrons, an incorporeal banshee-like creature, through a ritualistic act of slitting their own throat in ultimate devotion to their horrific masters."
+ 
+ helya:0 "Helism"
+ helya_adj:0 "Helic"
+ helya_adherent:0 "Helist"
+ helya_adherent_plural:0 "Helists"
+ helya_desc:0 "Helya, the matriarch of Helheim, which is the underworld of the Vrykul who have fallen and haven't ascended to the Halls of Valour, was a Watcher who assisted Odyn in imprisoning elementals into the Elemental Plane. She would be the first spectral Val'kyr when she rebelled against Odyn. She would soon create a champion who would bear Odyns Warswords of the Valajar, which Helya stole from Odyn. This champion would slay many for their souls to serve in Helheim. She would also create the Helarjar, a holy order mocking Odyn's Valarjar. The Kvaldir of Helya would also be responsible for murdering those of dishonour and unworth, taking them to Helheim."
```

Changed first and last sentence. Changed spectral Val'kyr sentence, and following two sentences.
```suggestion
 helya_desc:0 "Helya, the matriarch of Helheim, which is the underworld of the Vrykul who have fallen and failed to ascended to the Halls of Valour, was a Watcher who assisted Odyn in imprisoning elementals into the Elemental Plane. She would become the first spectral Val'kyr when she rebelled against Odyn. She would later create a champion who would bear Odyns Warswords of the Valajar, which Helya stole from Odyn. This champion would slay many so that their souls would serve in Helheim. She would also create the Helarjar, a holy order mocking Odyn's Valarjar. The Kvaldir of Helya would also be responsible for murdering the dishonourable and unworthy, taking them to Helheim."
```